subjects/reversemenuindex/README.md

```diff
@@ -0,0 +1,41 @@
+## reversemenuindex
+
+### Instructions
+
+The restaurant employees are having a really tough day and are delivering the customers' food in the wrong order. You need to fix the problem so that they can deliver it correctly.
+
+Write a function `ReverseMenuIndex()` that takes an array of strings as arguments and returns another array of strings with the menu in the correct order.
```

Comment:
Wouldn't it be a slice of strings?

Reply:
You are correct, thank you for this observation. It is fixed!